README.md

```diff
@@ -7,9 +12,11 @@
 
 **[Documentation](https://kaolin.readthedocs.io/en/latest/)** | **[Paper](https://arxiv.org/abs/1911.05063)**
 
+_Kaolin is an active work-in-progress, we are working toward a stable beta release. Please Watch our github repository for releases to be notified. To start trying our code today see instructions below._
```

Comment:
Can we color this in red and make boldface?

Reply:
I changed it for bold, coloring is not really possible in Markdown, if necessary I can make an image.